cuebot/src/main/java/com/imageworks/spcue/dao/postgres/DispatcherDaoJdbc.java

```diff
@@ -353,6 +355,24 @@ public boolean findUnderProcedJob(JobInterface excludeJob, VirtualProc proc) {
          }
     }
 
+    @Override
+    public boolean findHigherPriorityJob(JobDetail baseJob, VirtualProc proc) {
+        long start = System.currentTimeMillis();
+        try {
+            return getJdbcTemplate().queryForObject(
+                    FIND_HIGHER_PRIORITY_JOB_BY_FACILITY,
+                    Integer.class, baseJob.priority, proc.getFacilityId(),
```

Comment:
nit: Possible to use `Boolean.class` as the return type?

Reply:
done.